README.md

```diff
@@ -1 +1,93 @@
-# hapi-rate-limit
\ No newline at end of file
+# hapi-rate-limit
+A [Hapi](http://hapijs.com/) plugin that enables rate-limiting for GET, POST, and DELETE requests. This plugin can be configured with custom
+rates on a route-by-route basis.
+
+## Register the plugin
+```
+const Hapi = require('hapi');
+
+const server = new Hapi.Server();
+
+server.register([
+  register: require('../lib'),
+  options: {
+    defaultRate: {
+      limit: 10,
+      window: 60
+    },
+    redisClient: myThenRedisClient
+    overLimitError: () => ErrConstructor;
```

Comment:
Remove the `() =>` since this should be an error constructor, not a function that returns an error constructor.